v2/emailpassword/common-customizations/get-user-info.mdx

```diff
@@ -67,8 +76,14 @@ async function getUserInfo() {
 
 <TabItem value="go">
 
+<!-- END COPY SECTION -->
+
 You can get a user's information on the backend using the `^{getUserByEmailGo}` and `GetUserById` functions:
```

Comment:
this also needs to be added in the python code snippet

Reply:
Done.